css/index.css

```
@@ -22,6 +25,8 @@ body.dark {
   --field-borders: #222;
   --primary-button-bg: #eee;
   --primary-button-color: #111;
+  --delete-button-background: #dc3545;
+  --font-color-delete: #ccc;
```

```suggestion
  --delete-button-borders: #E76F7B;
  --font-color-delete: #E76F7B;
```

I changed the font color a bit to adjust the contrast and added a different border color in dark mode.